File: All Star Code Challenge #14 - Find the median.py
def median(array):
    arr_s = sorted(array)
    if len(arr_s)%2==0:
        return (arr_s[len(arr_s)//2]+arr_s[len(arr_s)//2-1])/2
    else:
        return arr_s[len(arr_s)//2]

File: All Star Code Challenge #14 - Find the median.ts
export function median(array: number[]):number {
  const arr_s = array.slice().sort((a,b) => a-b)
  if(array.length%2==0){
    return (arr_s[Math.floor(arr_s.length/2)]+arr_s[Math.floor(arr_s.length/2)-1])/2
  }else{
    return arr_s[Math.floor(arr_s.length/2)]
  }
}
